Created KeyBoardWordFinderController.

xedisu opened the pull request:
Created index page.
Created keyboardWordFinderInputPage.html
Created keyboardWordFinderOutputPage.html
Created WordsDTO.java.
Created Application.java.

CatalinDobrescu committed:
Created KeyBoardWordFinderController.
Created index page.
Created keyboardWordFinderInputPage.html
Created keyboardWordFinderOutputPage.html
Created WordsDTO.java.
Created Application.java.

BogdanTanase commented on src/main/java/com/algorithmtron/Application.java:
```diff
@@ -0,0 +1,13 @@
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+
```
Do you need this space?

xedisu replied:
You're right, fixed!

BogdanTanase commented on src/main/resources/templates/index.html:
```diff
@@ -0,0 +1,11 @@
+    <title>Algorithmtron 2023</title>
+</head>
+<body>
+<h4>Welcome to Algo!</h4>
```
Perhaps a nicer greetings?

BogdanTanase reviewed (changes requested):
Good job! Left some small comments :D

CatalinDobrescu committed:
Created KeyBoardWordFinderController.
Created index page.
Created keyboardWordFinderInputPage.html
Created keyboardWordFinderOutputPage.html
Created WordsDTO.java.
Created Application.java.
